lua/modules/creations.lua

```diff
@@ -430,7 +430,9 @@ creations.createModalContent = function(_, config)
 		creationsContent.node = grid
 
 		creationsContent.willResignActive = function(_)
-			grid:cancelRequestsAndTimers()
+			if grid and grid.cancelRequestsAndTimers then
```

Comment:
Note: grid may not still be there at the time `willResignActive` is called, as, creationsContent.node can change for something else